### PR TITLE
check for error when calling DownloadFile

### PR DESCRIPTION
**Pull request by mikewiacek**

### CODE REVIEW

**Comment by mikewiacek** [comment]
@googlebot I fixed it.

File: client.go [hunk 1]
```diff
@@ -276,7 +276,18 @@ func (cli *Client) DownloadFile(hash string, w io.Writer) (int64, error) {
...
+		return io.Copy(w, resp.Body)
+	}
+
+	// See if there is an error in the response.
+	if _, err := cli.parseResponse(resp); err != nil {
+		return 0, err
+	}
+
+	// Last resort return a generic error.
+	return 0, fmt.Errorf("vt: unknown error downloading %s", hash)
```
Comment (plusvic) [inline client.go]: Remove the `vt: ` prefix, for consistency with other error messages, also adding the HTTP error code to the response could be helpful.

Comment (mikewiacek) [reply client.go]: Done!

**Comment by mikewiacek** [comment]
PTAL

**Pull request merged**